### DISCUSSION TITLE
How do you open the warp GUI from console?

### DISCUSSION THREAD

**Discussion by tylerking** [Zrips/CMI#7225] [Q&A]
Hello. What is the command to open the warp GUI for a player from the console? I've tried everything I can think of but no luck. Any help would be appreciated. Cheers.

**Comment by davidgb8246** [comment]
You can sudo the player, the command that should open the gui by hand in game. Ex: `sudo doitliketyler warp-command`

**Comment by tylerking** [comment]
Thank for the response. Unfortunately that would require the player to have the permission cmi.command.warp which I do not want. I only want to be able to open the warp GUI from console for them. (Same for the scavenge GUI but for some reason that also doesn't seem to have a player argument)
